openhabian-setup.sh

```diff
@@ -102,6 +102,7 @@ if [[ -n "$UNATTENDED" ]]; then
   vimrc_copy
   install_tailscale "install" && setup_tailscale
   misc_system_settings
+  openhab_shell_interfaces
```

Comment:
Shouldn't this go after openHAB is installed?

Reply:
indeed, here you are